content/en/blog/2020/wasmhub-istio/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Extended and Improved WebAssemblyHub to Bring the Power of WebAssembly to Envoy and Istio
+subtitle: A hub to build, publish, share and deploy WebAssembly Envoy extensions.
```

Comment:
@bettyjunod also looks like the subtitle is _not_  supposed to have a period "." if you remove you should be good.